docs/content/Coding Milestones/PoC2023q1/example1-subs/example1-post-kcp.md

```diff
@@ -92,11 +92,66 @@ to pass the new file with `--kubeconfig` on the command lines when
 using kcp or KubeStellar.
 
 
+### Space Manager
+
+The Space Manager will use space provider to create and manage underlying cluster
+for each Space object. Space objects are created in `root:space-mgt` context
```

Comment:
(A) In English this parses as saying that the context is under the namespace.  I think that is the opposite of what you mean.  I suspect that what you mean would be better said as "Space objects are created in the `spaceprovider-default` namespace in the ... context".

(B) I assume that by "context" here you mean the concept in a kubeconfig file.  A context does not hold API objects or namespaces.  A context is a particular kube client configuration.  So I suspect that what you mean would be better written as "Space objects are created in the `spaceprovider-default` namespace using the context named `root:space-mgt` in your chosen kubeconfig file".

Reply:
why the name of the context in the kubeconfig file is so important that it needs to be mentioned? Is it because there is some convention that for context naming that the reader need to be aware of?

Reply:
well, the answer seems to be in the script below, where you need to set that context - ignore my comment